partner_code_program_purchase/tests/test_partner_code_program_purchase.py

```diff
@@ -0,0 +1,35 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestPartnerCodeProgramPurchase(TransactionCase):
```

Comment:
Can you add the test where you create a purchase and check if the purchase has the partner's code program?